ros/src/common/libs/amathutils_lib/include/amathutils_lib/kalman_filter.hpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/LU>
+
+class KalmanFilter
+{
+public:
+  KalmanFilter();
```

Comment:
@TakaHoribe  Can you please add comments to your header files? (doxygen format http://wiki.ros.org/rosdoc_lite)

Reply:
updated for doxygen format in 10a8a56.

Reply:
great!